templates/statefulset.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: StatefulSet

```

Comment:
Change to Deployment since we want shared data at the container level instead of each container being unique